pandas/tests/tools/test_numeric.py

```diff
@@ -381,3 +381,20 @@ def test_downcast_limits(self):
         for dtype, downcast, min_max in dtype_downcast_min_max:
             series = pd.to_numeric(pd.Series(min_max), downcast=downcast)
             assert series.dtype == dtype
+
+    def test_coerce_uint64_conflict(self):
+        # see gh-17007 and gh-17125
+        #
+        # Still returns float despite the uint64-nan conflict,
+        # which would normally force the casting to object.
```

Comment:
can you also test with ``errors='raise'`` and ``ignore`` here

Reply:
Done.